kubernetes/cray-postgres-operator/update_dependencies.sh

```diff
@@ -21,11 +21,11 @@ sha=$(git rev-parse --short HEAD)
 cd $return_dir
 cp -r /tmp/postgres-operator/charts ./
 
-docker pull dtr.dev.cray.com/craypc/chartsutil:latest
-chartsutil="docker run -it --rm -v $(pwd)/charts:/charts dtr.dev.cray.com/craypc/chartsutil:latest"
+docker pull artifactory.algol60.net/csm-docker/stable-local/craypc/chartsutil:latest
+chartsutil="docker run -it --rm -v $(pwd)/charts:/charts artifactory.algol60.net/csm-docker/stable-local/craypc/chartsutil:latest"
 $chartsutil helm package /charts/postgres-operator -d /charts/
```

Comment:
This isn't being done anymore... the subcharts aren't being tar'ed.

Reply:
Good point. Removed this file as it is no longer needed.